pccs/setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p /opt/intel/pccs/config/
+cd /opt/intel/pccs/config/
+
+if [ -n "$APIKEY" ]; then
+    apikey="$APIKEY"
+    sed -i "s/\"ApiKey\"[ ]*:[ ]*\"\",/\"ApiKey\": \"$apikey\",/" default.json
+else
+    echo "ERROR: You need to submit an APIKEY, otherwise your PCCS can not connect to the PCS"
+    exit 1
+fi
+
+if [ -n "$USERPASS" ]; then
+    userpass="$USERPASS"
+else
+    userpass="12345"
```

Comment:
```suggestion
```
I would prefer to not set any password if the user does not provide any, instead of a hardcoded one.